### ISSUE TITLE
A question about the gradient availability of a compiled model.

### ISSUE THREAD

**Issue by useongkim** [facebookincubator/AITemplate#135]
Hello, 

After model compilation, will the gradient computation through input tensors be available during inference time?

Typically, inference libraries do not support backpropagation through model parameters because the model parameters are fixed during the inference time. 

However, the backpropagation through input tensors during the inference time can still be helpful when we solve an inverse problem with a neural network-based forward model compiled by AITemplate.

Thank you.

Useong Kim.
